tests/lit-tests/explicit-vector.ispc

```diff
@@ -0,0 +1,85 @@
+// This test checks for the simplification of induction variables for loops and efficient
+// generation of accumulator widening/arithmetic.
+// See [GitHub issue ISPC/#2460](https://github.com/ispc/ispc/issues/2460) for details.
+
+// RUN: %{ispc} -DVECTOR=0 %s --nostdlib --emit-llvm-text --target=avx2-i32x4 --arch=x86-64 --debug-phase=296 --dump-file -o /dev/null
+// RUN: FileCheck %s --input-file ir_296_IndVarSimplifyPass.ll --check-prefix=CHECK_VECTOR
```

Comment:
As I suggested above, the final LLVM IR should be checked. If specific phase is checked, then the folder with test-specific name should be specified, otherwise it's a potential problem for concurrent test runs.

Reply:
See new test for indvars: `tests/lit-tests/ind-var-simplifiy.ispc`.